Eevee.Sleep.Bot/Utils/DiscordMessageMaker/DiscordMessageMakerForInGameAnnouncement.cs

```diff
@@ -40,4 +41,23 @@ public static class DiscordMessageMakerForInGameAnnouncement {
             .WithCurrentTimestamp()
             .Build();
     }
+
+    public static Embed MakeInGameAnnouncementUpdateMessage(InGameAnnouncementDetailModel detail) {
+        var content = detail.Text;
```

Comment:
Good to skip contents. I am gonna make the website to render the contents utilizing the API we have instead

Reply:
Make the API also refer to in-game announcement? Not all records are retrieved, but satisfied?

Reply:
Currently thinking the API to show the in-game announcement instead, or maybe another set of API endpoints 🤔